src/Views/Movies/Create.cshtml

```diff
@@ -0,0 +1,41 @@
+@using Microsoft.AspNetCore.Mvc.Localization
+@using System.Globalization
+@inject IViewLocalizer Localizer
+@model MvcMovie.Models.Views.MovieViewModel
+
+@{
+    ViewData["Title"] = Localizer["CreateTitle"];
+}
+
+<form asp-action="Create">
```

Comment:
its ok for now, but next time try to use [html helpers](http://www.tutorialsteacher.com/mvc/html-helpers)